src/OpenFga.Sdk/Client/Client.cs

```diff
@@ -0,0 +1,377 @@
+        CancellationToken cancellationToken = default) =>
+        await api.WriteAuthorizationModel(body, cancellationToken);
+
+    /**
+     * ReadAuthorizationModel - Read the current authorization model
+     */
+    public async Task<ReadAuthorizationModelResponse> ReadAuthorizationModel(
+        ClientReadAuthorizationModelOptions? options = default,
+        CancellationToken cancellationToken = default) {
+        var authorizationModelId = GetAuthorizationModelId(options);
+        if (authorizationModelId == null) {
+            throw new FgaRequiredParamError("ClientConfiguration", "AuthorizationModelId");
+        }
+
+        return await api.ReadAuthorizationModel(authorizationModelId, cancellationToken);
+    }
+
+    /**
+     * ReadLatestAuthorizationModel - Read the latest authorization model for the current store
+     */
+    public async Task<ReadAuthorizationModelResponse> ReadLatestAuthorizationModel(
+        IClientRequestOptionsWithAuthZModelId options = default,
+        CancellationToken cancellationToken = default) {
+        var response =
+            await ReadAuthorizationModels(new ClientReadAuthorizationModelsOptions {PageSize = 1}, cancellationToken);
+        return new ReadAuthorizationModelResponse {AuthorizationModel = response.AuthorizationModels?[0]};
+    }
+
+    /***********************
+     * Relationship Tuples *
+     ***********************/
+
+    /**
+     * Read Changes - Read the list of historical relationship tuple writes and deletes
+     */
+    public async Task<ReadChangesResponse> ReadChanges(ClientReadChangesRequest body,
+        ClientReadChangesOptions? options = default,
+        CancellationToken cancellationToken = default) =>
+        await api.ReadChanges(body.Type, options?.PageSize, options?.ContinuationToken, cancellationToken);
+
+    /**
+     * Read - Read tuples previously written to the store (does not evaluate)
+     */
+    public async Task<ReadResponse> Read(ClientReadRequest body, ClientReadOptions? options = default,
+        CancellationToken cancellationToken = default) =>
+        await api.Read(
+            new ReadRequest {
+                TupleKey = body, PageSize = options?.PageSize, ContinuationToken = options?.ContinuationToken
+            }, cancellationToken);
+
+    /**
+     * Write - Create or delete relationship tuples
+     */
+    public async Task<ClientWriteResponse> Write(ClientWriteRequest body, ClientWriteOptions options = default,
+        CancellationToken cancellationToken = default) {
+        var maxPerChunk =
+            options.Transaction?.MaxPerChunk ??
+            1; // 1 has to be the default otherwise the chunks will be sent in transactions
+        var waitTimeBetweenChunksInMs =
+            options.Transaction?.WaitTimeBetweenChunksInMs ?? DEFAULT_WAIT_TIME_BETWEEN_CHUNKS_IN_MS;
+        var authorizationModelId = GetAuthorizationModelId(options);
+
+        if (options.Transaction?.Disable != true) {
```

Comment:
Just want to verify whether this is different than the JS SDK case where it is not transaction unless you specifically ask for transaction

Reply:
Isn't it the same here?

If `options.Transaction?.Disable` is anything but true (as in so long as you are not disabling the transaction), it will run in a transaction mode.